packages/ag-charts-enterprise/src/series/treemap/treeMapLabelFormatter.test.ts

```diff
@@ -0,0 +1,218 @@
+import { _Scene } from 'ag-charts-community';
+
+import {
+    formatSingleLabel,
+    formatStackedLabels,
+    generateLabelSecondaryLabelFontSizeCandidates,
+    maximumValueSatisfying,
+} from './treeMapLabelFormatter';
+
+const { Text } = _Scene;
+
+const wrap = jest.fn();
+const computeBBox = jest.fn();
+
+Text.wrap = wrap;
+Text.prototype.computeBBox = computeBBox;
```

Comment:
Nit: Could you perform these mock setups in a `before()` or `beforeEach()` to avoid affecting other test cases? Maybe inside the describe block (along with moving the `afterEach()`?